src/Sulu/Bundle/PreviewBundle/Resources/js/package.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "sulu-preview-bundle",
+    "description": "Adds support for preview to Sulu",
+    "license": "MIT",
+    "repository": "https://github.com/sulu/sulu.git",
+    "dependencies": {
+        "debounce": "^1.0.2",
+        "mobx-react": "^5.0.0",
+        "query-string": "^6.0.0",
+        "react": "^15.6.0",
+        "react-dom": "^16.2.0",
+        "sulu-admin-bundle": "file:../../../AdminBundle/Resources/js"
```

Comment:
We have changed the handling of dependencies: stuff like `mobx`, `mobx-react`, `react`, `react-dom` (do you even really need that one in here?) are now `peerDependencies`.